cpp/mcap/include/mcap/writer.hpp

```diff
@@ -17,10 +17,21 @@ namespace mcap {
  */
 struct McapWriterOptions {
   /**
-   * @brief Disable CRC calculations for Chunks, Attachments, and the Data and
-   * Summary sections.
+   * @brief Disable CRC calculations for Chunks.
    */
-  bool noCRC = false;
+  bool noChunkCRC = false;
+  /**
+   * @brief Disable CRC calculations for Attachments.
+   */
+  bool noAttachmentCRC = false;
+  /**
+   * @brief Enable CRC calculations for all records in the data section.
+   */
+  bool enableDataCRC = false;
+  /**
+   * @brief Disable CRC calculations for the summary section.
+   */
+  bool noSummaryCRC = false;
```

Comment:
I assume https://github.com/ros-tooling/rosbag2_storage_mcap/blob/main/rosbag2_storage_mcap/src/mcap_storage.cpp#L117-L132 needs to change as well once this PR lands?